docs/plugins/irods_rule_language.md

```diff
@@ -586,6 +586,9 @@ This Is Not A Valid Function Name or Rule Name
 
 ### Rule Condition
 
+!!! Note
+    **Use of `on()` may lead to unexpected behavior!** If all rule conditions for a rule fail, the rule will return an error! Use `if()` for simple conditions.
```

Comment:
Something about `for simple conditions.` doesn't sit right with me.

That phrase implies there's a way to exactly identify a simple condition vs a complex condition when in fact that is subjective.

What if we just said - "Prefer using `if()` instead of `on()`."

Thoughts?

Reply:
i had changed the wording up above in my suggestion.   search for 'familiar'...

Reply:
The wording you provided is good too. I'm fine with using that.

Reply:
Oops, missed the additional wording change above. Added!